BUILDING.md

```diff
@@ -4,12 +4,15 @@
 * Windows:
   * [Docker Desktop](https://docs.docker.com/desktop/setup/install/windows-install/)
   * make
+  * go >= 1.23.10
```

Comment:
I wonder if you should not just link to the go.mod file to avoid managing the sync and be at some point desynchronize, wdyt?
```suggestion
  * go >= check version used in the [go.mod](go.mod) file
```

Reply:
Seems like a good idea. I updated it to link to `go.mod`